models/public/wav2vec2-base/accuracy-check.yml

```diff
@@ -2,7 +2,6 @@ models:
   - name: wav2vec2-base
     launchers:
       - framework: openvino
-        allow_reshape_input: true
```

Comment:
please keep AC config, it is still used in internal infra for validation models with static shape